tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -344,12 +344,79 @@ int main(int argc, char **argv)
 
             EXPECT_NOT_EQUAL(conn->psk_params.psk_list.mem.allocated, 0);
             EXPECT_EQUAL(conn->psk_params.psk_list.len, S2N_TEST_PSK_COUNT);
+            EXPECT_NULL(conn->psk_params.chosen_psk);
 
             EXPECT_SUCCESS(s2n_tls13_handle_handshake_secrets(conn));
 
-            /* Verify PSKs are wiped */
-            EXPECT_EQUAL(conn->psk_params.psk_list.mem.allocated, 0);
-            EXPECT_EQUAL(conn->psk_params.psk_list.len, 0);
+            /* Verify secrets are wiped */
+            for (size_t i = 0; i < conn->psk_params.psk_list.len; i++) {
+                struct s2n_psk *psk;
+                EXPECT_OK(s2n_array_get(&conn->psk_params.psk_list, i, (void**)&psk));
+                EXPECT_NOT_NULL(psk->identity.data);
+                EXPECT_NULL(psk->secret.data);
+                EXPECT_EQUAL(psk->secret.size, 0);
```

Comment:
Did any of your PSKs have secrets to be wipe? It looks like you never set any secrets